lib/bundler/audit/cli.rb

```diff
@@ -34,6 +34,7 @@ class CLI < ::Thor
       method_option :verbose, :type => :boolean, :aliases => '-v'
       method_option :ignore, :type => :array, :aliases => '-i'
       method_option :update, :type => :boolean, :aliases => '-u'
+      method_option :db_path, :type => :string, :aliases => '-p', default: nil
```

Comment:
Having the proper default here vs `nil` is a nicer way, it avoids passing around nil.

Reply:
@pxlpnk are you suggesting? 

```ruby
default: File.join(Dir.home, ".local", "share", "ruby-advisory-db")
```

```irb
irb(main):001:0> File.join(Dir.home, '.local', 'share', 'ruby-advisory-db')
=> "/home/mokha/.local/share/ruby-advisory-db"
```

Reply:
I think that could work.  Or use a variable and assign it that value.

Reply:
@mokhan @pxlpnk updates pushed.